package.json

```diff
@@ -48,6 +48,10 @@
   },
   "devDependencies": {
     "@alaskaairux/orion-web-core-style-sheets": "^0.3.511780638",
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
+    "@babel/node": "^7.2.2",
+    "@babel/preset-env": "^7.4.3",
```

Comment:
You can remove the babel 6 dependencies below: `babel-cli`, `babel-core`, `babel-preset-es2015`.

Reply:
Yeah, lazy code post update cleanup. Thanks!